cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -327,14 +336,17 @@ class device_row_comparator {
 
     template <typename Element,
               CUDF_ENABLE_IF(not cudf::is_relationally_comparable<Element, Element>() and
-                             not std::is_same_v<Element, cudf::struct_view>)>
+                             not std::is_same_v<Element, cudf::struct_view> and
+                             not std::is_same_v<Element, cudf::list_view>)>
```

Comment:
https://github.com/rapidsai/cudf/blob/c11b780d6e09d6e6ad539c5a1c14d14393737c9d/cpp/include/cudf/utilities/traits.hpp#L696-L699
```suggestion
                             not cudf::is_nested<Element>())>
```
(Call me a trait-or if you must.)